src/OrchardCore.Modules/OrchardCore.Taxonomies/Manifest.cs

```diff
@@ -19,5 +19,6 @@
     Id = "OrchardCore.Taxonomies.ContentsAdminList",
     Name = "Taxonomies Contents List Filters",
     Description = "Provides taxonomy filters in the contents list.",
+    Dependencies = new[] { "OrchardCore.Taxonomies" },
```

Comment:
Why we need to self dependency?!!

Reply:
It's not a self dependency, this is the `OrchardCore.Taxonomies.ContentsAdminList` feature that depends on the `OrchardCore.Taxonomies` feature

Reply:
My thought while the feature in the same module no need to reference another feature in the same module

Reply:
System ones at the top, others by alphabetic order, so Fluid before OC ones

I'm using under VStudio the `Remove and Sort Usings` cmd when you right click on the usings

Or `ctrl + RG` through the keyboard.

Reply:
A feature may depend (or not) on another feature of the same module

We do that in many other modules

Reply:
Ok, thanks